lib/rules/focus.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "focus",
+  "selector": "[onFocus]",
+  "tags": [
+    "wcag2a",
+    "wcag211",
+    "cat.keyboard"
+  ],
+  "metadata": {
+    "description": "Ensure that executes code onFocus do not loose it when code is executed",
+    "help": "Elements should not loose focus when it just gained it"
+  },
```

Comment:
`lose`

Reply:
I'd rewrite this as "Elements should not blur on focus"